tests/unit/utils/test_config_mlflow.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+import pytest
+
+from lineapy.utils.config import DEFAULT_ML_MODELS_STORAGE_BACKEND, options
+from tests.util import clean_lineapy_env_var
+
+mlflow = pytest.importorskip("mlflow")
```

Comment:
not needed for this set of tests?

Reply:
I add this test to check how the `default_ml_models_storage_backend` behaves if we set the `mlflow_tracking_uri`. Because of the mlflow dependency, I need separate it from all other config tests.